src/rc.js

```diff
@@ -1 +1,4 @@
-module.exports = require('rc')('DeltaChat', {})
+module.exports = require('rc')('DeltaChat', {
+  'log-debug': false,
+  'log-to-console': false
+})
```

Comment:
@Simon-Laux This is technically not needed since they are falsey it will not matter, but it shows which properties you _can_ configure.